rusoto/core/src/region.rs

```diff
@@ -15,21 +15,52 @@ use std::fmt::{Display, Error as FmtError, Formatter};
 /// `CnNorth1` is currently untested due to Rusoto maintainers not having access to AWS China.
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub enum Region {
+    /// Server for North-East part of America
```

Comment:
I don't think the word server makes sense here.

Perhaps something like:
```
The Region that lines up with the North-East part of the US.
```

or even:

```
The Region that covers the North-East part of the US.
````